src/olympia/files/tests/test_utils_.py

```diff
@@ -325,6 +325,25 @@ def test_handle_utf_bom(self):
         parsed = utils.ManifestJSONExtractor(None, manifest).parse()
         assert parsed['name'] == '...'
 
+    def test_comments_are_allowed(self):
+        json_string = """
+        {
+            // Required
+            "manifest_version": 2,
+            "name": "My Extension",
+            "version": "versionString",
+
+            // Recommended
+            "default_locale": "en",
+            "description": "A plain text description"
+        }
+        """
+        manifest = utils.ManifestJSONExtractor('/fake_path',
```

Comment:
I would indent it more like

```python
manifest = utils.ManifestJSONExtractor(
    '/fake_path', json_string).parse()
```

or even split the instantiation of `ManifestJSONExtractor` and the `parse()` call to make it even easier to read. Also, only nitpicks.